templates/definition/meter/fox-ess-h1.yaml

```diff
@@ -8,8 +8,7 @@ params:
   - name: usage
     choice: ["pv", "battery"]
   - name: modbus
-    choice: ["rs485", "tcpip"]
-    id: 247
```

Comment:
ID raus?

Reply:
Nee, die gehört da schon hin. Das ist die Fox-Standard-ID.
Gerade erst eingebaut.

Reply:
Sind drin. Bei rs485 brauchts die nicht?

Reply:
Doch, in allen Fällen.

Reply:
Andersrum: Ich hab sie dort eingebaut :)

Reply:
> Bei rs485 brauchts die nicht?

Du hast das rs485 Template ausgebaut. Dann brauchts das hier auch nciht mehr und damit auch keine id.

Reply:
Fixed